tests/Doctrine/Tests/CouchDB/Functional/CouchDBClientTest.php

```diff
@@ -683,4 +684,53 @@ public function testEncodeQueryParamsCorrectly()
 
         $this->assertEquals(0, $result->getTotalRows());
     }
+
+
+    public function testBulkGet()
+    {
+        $client = $this->couchClient;
+
+
+        $docs = [];
+        $expectedRows = [];
+        foreach (range(1, 3) as $i) {
+            list($id, $rev) = $client->postDocument(['foo' => 'bar'.$i]);
```

Comment:
```suggestion
            [$id, $rev] = $client->postDocument(['foo' => 'bar'.$i]);
```

Reply:
The composer constraint is currently `"php": ">=5.4"`, but I assume we are going to have at least 7.2 soon.

Reply:
The composer constraint is currently `"php": ">=5.4"`, but I assume we are going to have at least 7.2 soon.